README.md

```diff
@@ -189,16 +209,18 @@ const event = MParticle.CommerceEvent.createImpressionEvent([impression])
 MParticle.logCommerceEvent(event)
 ```
 
-**Logging** screen events:
+To log screen events:
 
 ```js
 MParticle.logScreenEvent('Test screen', { 'Test key': 'Test value' })
 ```
 
 ## User
-**Setting** user attributes and tags:
 
-Use Identify or currentUser to retrieve the userID for these calls
+Get the current user in order to apply and remove attributes, tags, etc.
+
+Use `Identify` or `currentUser` methods to retrieve the `userID` for these calls
```

Comment:
Seems like the react api looks different from the Flutter one here. This should be:

```suggestion
To set, remove, and get user details, call the `User` or `Identity` methods as follows:
```